client/packages/common/src/styles/theme.ts

```diff
@@ -190,16 +190,16 @@ export const themeOptions = {
   },
   typography: {
     body1: {
-      fontFamily: 'InterVariable',
+      fontFamily: 'Inter Variable',
```

Comment:
Oooh, was that wrong the whole time ?

Reply:
no - they've annoyingly updated the name in the new version 🙄